project/Deps.scala

```diff
@@ -5,41 +5,41 @@ object Deps {
 
   object V {
     val bouncyCastle = "1.70"
-    val dropwizardMetricsV = "4.2.7" //https://github.com/dropwizard/metrics
-    val logback = "1.2.10"
+    val dropwizardMetricsV = "4.2.8" //https://github.com/dropwizard/metrics
+    val logback = "1.2.11"
     val grizzledSlf4j = "1.3.4"
     val scalacheck = "1.15.4"
     val scalaTest = "3.2.10"
 
     val scalaTestPlus =
       "3.2.2.0" //super annoying... https://oss.sonatype.org/content/groups/public/org/scalatestplus/
-    val slf4j = "1.7.33"
+    val slf4j = "1.7.36"
     val spray = "1.3.6"
     val zeromq = "0.5.2"
-    val akkav = "10.2.7"
+    val akkav = "10.2.9"
     val playv = "2.9.2"
     val akkaStreamv = "2.6.18"
     val jUnixSocketV = "2.4.0"
     val scodecV = "1.1.30"
     val junitV = "0.13.3"
     val nativeLoaderV = "2.4.0"
-    val typesafeConfigV = "1.4.1"
+    val typesafeConfigV = "1.4.2"
 
     val scalaFxV = "17.0.1-R26"
     val javaFxV = "18-ea+10"
 
     val asyncNewScalaV = "1.0.1"
 
     val flywayV = "6.4.2"
-    val postgresV = "42.3.1"
+    val postgresV = "42.3.3"
     val akkaActorV = akkaStreamv
     val slickV = "3.3.3"
     val sqliteV = "3.36.0.3"
 
     val scalameterV = "0.17"
     val scalamockV = "5.2.0"
     val scalaCollectionCompatV = "2.6.0"
-    val pgEmbeddedV = "0.13.4"
+    val pgEmbeddedV = "1.0.0"
```

Comment:
This needs to be reverted. They are moving to a docker based postgres environment in the `1.x.x` series. We do not have that setup. 

https://github.com/opentable/otj-pg-embedded/issues/166

Reply:
https://github.com/bitcoin-s/bitcoin-s/runs/5634927314?check_suite_focus=true#step:6:2672